lib/types/json.js

```diff
@@ -77,10 +77,13 @@ function json(options) {
 
     if (strict) {
       var first = firstchar(body)
-
+      var message = 'Unexpected token ' + first
       if (first !== '{' && first !== '[') {
+        if (process.version[1] >= 6) {
+          message += ' in JSON at position 1'
```

Comment:
Why is the position static? The character can be at any position, and this message would be confusing when it tells you that a double quote character is unexpected at position 1, but the first quote is 20 characters from the start of the string.